github-actions/pulumi-nodejs.yml

```
@@ -41,23 +41,39 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Use Node.js
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v2
         with:
           node-version: 12.x
 
-      - name: Install Pulumi CLI
-        uses: pulumi/action-install-pulumi-cli@v1.0.1
-
-      - name: Restore npm dependencies
-        run: |
-          npm --prefix $PULUMI_WORKING_DIRECTORY ci
-          # If you are using yarn instead with Pulumi, comment the above line and uncomment the below line.
-          # yarn install --cwd $PULUMI_WORKING_DIRECTORY
+      - run: npm install
```

Let's use `ci` instead of the `install` command when restoring node deps.

```suggestion
      - run: npm ci
```